File: frameworks/Java/baratine/src/main/java/testTechempowerBaratine/PlaintextService.java
package testTechempowerBaratine;

import io.baratine.service.Result;
import io.baratine.service.Service;
import io.baratine.web.Get;

@Service
public class PlaintextService
{
  @Get("/plaintext")
  public void hello(Result<String> result)
  {
    result.ok("Hello, World!");
  }
}

File: frameworks/TypeScript/feathersjs/src/services/plaintext/plaintext.service.ts
// Initializes the `plaintext` service on path `/plaintext`
import { ServiceAddons } from '@feathersjs/feathers';
import { Application } from '../../declarations';
import { Request, Response } from 'express';

// Add this service to the service type index
declare module '../../declarations' {
  interface ServiceTypes {
    'plaintext': ServiceAddons<any>;
  }
}

export default function (app: Application): void {
  app.get('/plaintext', (req: Request, res: Response) => {
    res.setHeader('Content-Type', 'text/plain');
    res.send('Hello, World!');
  });
}
